charts/kargo/templates/argocd/role-binding.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: kargo-controller
-  namespace: {{ .Values.controller.argocd.namespace }}
+  namespace: {{ .Values.controller.argocd.namespace | default "argocd" }}
```

Comment:
This is already defaulted to `argocd` in `values.yaml`, but problems occur if a user overrides it to `""`.

Go code will default to `argocd` if this is unspecified...

The problem this change actually avoids is the resource being created in the cluster's _default_ namespace when this value is unspecified.

Reply:
Same comment applies in several other places.